frontend/src/components/landing/Plans.module.scss

```diff
@@ -50,18 +51,18 @@
   }
 
   .ribbon {
+    top: -10px;
+    right: -10px;
+    padding-left: $spacing-sm; /* The ribbon text indent */
     position: absolute;
-    margin-left: 250px;
-    margin-top: -40px;
-    width: 180px;
+    width: 126px;
```

Comment:
From the face of it, this is still a magic number, so might be good to add a comment to this one that it's the width of the ribbon background image?

Reply:
Will do!